src/main/java/frc/robot/Constants.java

```diff
@@ -28,35 +28,78 @@
+  public static final class ManipulatorConstants {
+
     // CAN IDs for the Intake Motors
-    public static final int INTAKE_MOTOR_RIGHT = 48;
-    public static final int INTAKE_MOTOR_LEFT = 47;
+    public static final int FEEDER_MOTOR = 17;
+    public static final int INTAKE_MOTOR = 25;
+    public static final int SHOOTER_MOTOR_RIGHT = 15;
+    public static final int SHOOTER_MOTOR_LEFT = 16;
 
     // CAN ID for the Pivot Motors
-    public static final int PIVOT_MOTOR = 13;
-
-    // Speed Constants for Intake and Pivot
-    public static final double INTAKE_SPEED = 0.6;
-    public static final double SCORE_HIGH_SPEED = -0.6; //UNTESTED VALUES PLEASE SET ACCORDING TO TEST
-    public static final double SCORE_MID_SPEED = -0.3;
-    public static final double SCORE_LOW_SPEED = -0.15;
+    public static final int PIVOT_MOTOR_RIGHT = 13;
+    public static final int PIVOT_MOTOR_LEFT = 14;
+
+    // Speed Constants for wheels
+    public static final double SHOOT_MAX_VEL_SET = 6630.0;
+    public static final double OUTTAKE_SPEED = -0.5;
+    public static final double INTAKE_SPEED = 0.5;
+    public static final double SCORE_HIGH_SPEED = SHOOT_MAX_VEL_SET * 0.6; //0.6
+    public static final double SCORE_MID_SPEED = SHOOT_MAX_VEL_SET * 0.3; //0.3
+    public static final double SCORE_LOW_SPEED = SHOOT_MAX_VEL_SET * 0.15; //0.15
     public static final double INTAKE_REST_SPEED = 0.00;
-    public static final double PIVOT_UP_SPEED = 0.2;
-    public static final double PIVOT_DOWN_SPEED = -0.2;
+    public static final double SHOOTER_kFF = 0.000082;
+    public static final double AMP_SPEED = 0.2;
+
+    // Constants for Pivot
+    public static final double PIVOT_UP_SPEED = 0.1;
+    public static final double PIVOT_DOWN_SPEED = -0.1;
     public static final double MOTOR_ZERO_SPEED = 0;
+    public static final double MOTOR_GRAV_SPEED = 0.02;
+    public static final double MOTOR_LOADBACK_SPEED = -0.1; //-0.07
+    public static final double MOTOR_LOADING_SPEED = 1; //0.07
+
+    // PID constants for pivot
+    public static final double PIVOT_kP = 0.23; //VOLTS
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0.0;
+    public static final double PIVOT_kS = 0.5;
+    public static final double PIVOT_kG = 0.25;
+    public static final double PIVOT_kV = 0.25; //3.45 default
+    public static final double PIVOT_kA = 0.01;
 
     // Encoder Values for the Intake and Pivot
-    public static final double PIVOT_MAX = 10;
-    public static final double PIVOT_MIN = 0;
+    public static final double PIVOT_MAX = -(Math.PI/2);
+    public static final double PIVOT_MIN = 0.08;
+    public static final double PIVOT_CLOSE_SCORE = -0.2;
+    public static final double PIVOT_FAR_SCORE = -0.56;
+    public static final double PIVOT_AMP_GOAL = -(Math.PI/2);
+    public static final double PIVOT_TRAP_SCORE = -0.15;
+    public static final double PIVOT_WEIGHT = 4.9; //kgs, dust 9.55, roomba 4.9
```

Comment:
why are we not using the dustpan pivot weight? we should remove this comment, since main brain will be just "dustpan" we can remove this comment